codeceptConfigHelper.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
     inventoryAPI: './tests/pages/api/inventoryAPI.js',
     mysqlTableDetailsPage: './tests/pages/mysqlTableDetailsPage.js',
     leftNavMenu: './tests/pages/leftNavMenu.js',
-    links: './linksHelper.js',
+    links: './tests/helper/linksHelper.js',
```

Comment:
@nazarovvi any reason why you want this to be a helper?

Reply:
Tidiness mainly. All _something_Helper.js belong in the helpers folder. Easier to find